.github/renovate.json5

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "github>indykite/.github//.github/renovate.json",
+  ],
+  "postUpdateOptions": [
+    "gomodTidy",
+    "gomodUpdateImportPaths"
+  ]
```

Comment:
Should we add similar setting like we have in `jarvis-proto` for grouping Go/CI/... minor+patch updates etc?